src/platform/packages/shared/response-ops/alerts-apis/hooks/use_get_muted_alerts_query.tsx

```diff
@@ -54,5 +54,8 @@ export const useGetMutedAlertsQuery = (
       }
     },
     enabled: ruleIds.length > 0 && enabled !== false,
+    refetchOnMount: false,
+    refetchOnWindowFocus: false,
+    refetchOnReconnect: false,
```

Comment:
I wonder if this should be the default config for react-query. @elastic/response-ops